cypress/integration/inventory/keyboard-shortcuts.spec.js

```diff
@@ -17,18 +24,40 @@ describe('ui-inventory: keyboard shortcut', () => {
     cy.visit(TopMenu.inventoryPath);
   });
 
-  after('Delete all data', () => {
+  afterEach('Delete all data', () => {
     cy.deleteUser(userId);
   });
 
   it('C345297 Keyboard Shortcut. Access to drop down menu', { tags: [testTypes.smoke] }, () => {
-    InventoryMainButton.verifyInventoryDropdownIsShown('false');
+    InventoryKeyboardShortcuts.verifyInventoryDropdownIsShown('false');
 
-    InventoryMainButton.openInventoryMenu();
-    InventoryMainButton.verifyInventoryDropdownIsShown('true');
-    InventoryMainButton.openShortcuts();
+    InventoryKeyboardShortcuts.openInventoryMenu();
+    InventoryKeyboardShortcuts.verifyInventoryDropdownIsShown('true');
+    InventoryKeyboardShortcuts.openShortcuts();
 
-    InventoryMainButton.waitModalLoading('Keyboard shortcuts');
-    InventoryMainButton.verifyInventoryDropdownIsShown('false');
+    InventoryKeyboardShortcuts.waitModalLoading('Keyboard shortcuts');
+    InventoryKeyboardShortcuts.verifyInventoryDropdownIsShown('false');
+  });
+
+  it('C345298 Keyboard Shortcut. Test the functionality of the different shortcut keys', { tags: [testTypes.smoke] }, () => {
+    InventoryKeyboardShortcuts.verifyInventoryDropdownIsShown('false');
+
+    InventoryKeyboardShortcuts.openInventoryMenu();
+    InventoryKeyboardShortcuts.openShortcuts();
+
+    InventoryKeyboardShortcuts.waitModalLoading('Keyboard shortcuts');
+    InventoryKeyboardShortcuts.pressHotKey(hotKeys.create);
+    InventoryKeyboardShortcuts.closeShortcuts();
+    InventoryKeyboardShortcuts.fillInstanceInfo();
```

Comment:
it's not fill. method contain saving additionally. Rename method or move pressure to save into separated step